content/en/docs/concepts/storage/volumes.md

```diff
@@ -657,7 +657,7 @@ metadata:
 spec:
   capacity:
     storage: 100Gi
-  # volumeMode field requires BlockVolume Alpha feature gate to be enabled.
+  # volumeMode field requires BlockVolume feature gate to be enabled.
```

Comment:
we can probably remove it altogether since it's been enabled by default for a while.